frame_capture_extension/background.js

```diff
@@ -0,0 +1,27 @@
+// listen for our browerAction to be clicked
+browser.browserAction.onClicked.addListener(tab => {
```

Comment:
If I load the extension, the extension background page shows error. 
> Uncaught ReferenceError: browser is not defined

Reply:
Seems Chrome doesn't have the `browser` namespace. It calls it `chrome` instead. I thought since the WebExtensions API is standard, Chrome should provide `browser` too. (I'm testing on Firefox). Will test on chrome and push the change.